bin/README.md

```diff
@@ -79,4 +79,20 @@ Why to deploy my data?
     * Can integrate access control (?)
 * Deploys a data API
 * [Other integrations e.g. push to google spreadsheets]
-* Dashboard showing your DataHub/Portal projects
\ No newline at end of file
+* Dashboard showing your DataHub/Portal projects
+
+### Extra supported commands 
+You can directly specify a folder to build from:
+```bash
+portal show path=\some\path\dataset 
```

Comment:
@risenW any reason we're using backslash? I think by default we use unix based systems.

Reply:
@anuveyatsu Just a typo from my end. Will update it now.

Reply:
Updated now